index.js

```diff
@@ -0,0 +1,82 @@
+const { exec } = require("child_process");
+const fs = require('fs');
+/**
+ * This function leverages the is-my-node-vulnerable package
+ * (https://github.com/RafaelGSS/is-my-node-vulnerable/blob/main/index.js) and
+ * extends it to be able to suppress / expire suppressions for pipeline use.
+ */
+function scanNode(){
+    exec('npx is-my-node-vulnerable@1.4.1',(error, stdout, stderr) => {
+        
+        // is-my-node-vulnerable module treats vulnerable node scans as errors,
+        // therefore we have to parse this output and come to our own conclusion
+        // whether to pass/fail based on ignored vulnerabilities (if any).
+        if (stderr) {
+
+            // Get every CVE vulnerability blurb along with patched versions
+            // from is-my-node-vulnerable output
+            pattern = /CVE-\d{4}-\d{4,}.+?Patched versions.+?\n/gs
+            var output = stderr.match(pattern);
+
+            validVulns = [];
+
+            const ignoreFile = './nodescan.json'
```

Comment:
Not worth doing in this PR, but if we start seeing more developers use this package, we should parameterize the `ignoreFIle` and allow users to specify a different path.

Reply:
Definitely agree